Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/MonthFootersDemoViewController.swift

```diff
@@ -0,0 +1,93 @@
+//  Created by Sebastien Audeon on 11/26/20.
+//  Copyright © 2020 Airbnb. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import HorizonCalendar
+
+class MonthFootersDemoViewController: DemoViewController {
```

Comment:
```suggestion
final class MonthFootersDemoViewController: DemoViewController {
```

Reply:
(small nit, I tend to use `final` for everything unless we're trying to write something subclass-able.

Reply:
Agreed!